net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
Personally I prefer a new option disabled to be added.  I have configs that is not in use at the moment but I do not want it to removed from the uci file either so that I can reenable it easily later.  This can also be useful for trying things up